src/main/webapp/i18n/de/notification.json

```diff
@@ -40,6 +40,98 @@
             "created": "Eine neue Mitteilung wurde mit ID {{ param }} erstellt",
             "updated": "Eine Mitteilung mit ID {{ param }} wurde geändert",
             "deleted": "Eine Mitteilung mit ID {{ param }} wurde gelöscht"
+        },
+        "groupNotification": {
+            "title": {
+                "attachmentChange": "Anhang aktualisiert",
+                "exerciseReleased": "Aufgabe veröffentlicht",
+                "exercisePractice": "Aufgabe offen zum Üben",
+                "quizExerciseStarted": "Quiz gestartet",
+                "liveExamExerciseUpdate": "DELive Exam Exercise Update",
+                "exerciseUpdated": "Aufgabe aktualisiert",
+                "programmingTestCasesChanged": "Testfälle für Programmieraufgaben geändert",
+                "newManualFeedbackRequest": "Eine neue Anfrage zu manuellem Feedback wurde eingereicht",
+                "duplicateTestCase": "Doppelte Testfälle wurden gefunden.",
+                "illegalSubmission": "Unzulässige Einreichung eines Studenten",
+                "newExercisePost": "Neuer Aufgabenbeitrag",
+                "newLecturePost": "Neuer Vorlesungsbeitrag",
+                "newCoursePost": "Neuer kursweiter Beitrag",
+                "newAnnouncementPost": "Neue Ankündigung",
+                "newReplyForExercisePost": "Neue Antwort auf Aufgabenbeitrag",
+                "newReplyForLecturePost": "Neue Antwort auf Vorlesungsbeitrag",
+                "newReplyForCoursePost": "Neue Antwort auf Kursbeitrag",
+                "courseArchiveStarted": "Kursarchivierung gestartet",
+                "courseArchiveFinished": "Kursarchivierung beendet",
+                "courseArchiveFailed": "Kursarchivierung fehlgeschlagen",
+                "examArchiveStarted": "Klausurarchivierung gestartet",
+                "examArchiveFinished": "Klausurarchivierung beendet",
+                "examArchiveFailed": "Klausurarchivierung fehlgeschlagen"
+            },
+            "text": {
+                "attachmentChange": "Anhang \"{{ placeholderValues.0 }}\" wurde aktualisiert.",
```

Comment:
Add CourseTitle and Exercise/Lecture Title

Reply:
I added them.